openshift/templates/rails-postgresql-persistent.json

```diff
@@ -138,7 +138,7 @@
           }
         ],
         "postCommit": {
-          "script": "bundle exec rake test"
+          "script": "/usr/bin/bundle exec rake test"
```

Comment:
Great find @yselkowitz thanks for researching this one

Reply:
Testing on my PR on sample and will report here

Reply:
Now on https://github.com/openshift/cluster-samples-operator/pull/408 rail test is not failing. Still see some other errors but this fix seems to be working